ErrorCodes_V2.md

```diff
@@ -0,0 +1,82 @@
+# AMPnet API Error Response Docs
+
+# Introduction
+
+AMPnet backend returns error response when errors happen (duh). 
+Error response contains:
+* `err_code` - structured in a way that should be easy to read and identify on-the-fly. 
+* `description` - defined in the tables below
+* `message` - generated depending on the problem and provides more information.
```

Comment:
One thing that I learned from my mentor while writing the bachelor thesis is that listing things in bullets is equal to the usual listing in a sentence.

So this:
```
A day consists of: morning, afternoon, evening, and night.
```

... is equal to this:
```
A day consists of: 
* morning, 
* afternoon, 
* evening, and
* night.
```

So punctuations should be preserved as it was a normal sentence.

```suggestion
Error response contains:
* `err_code` - structured in a way that should be easy to read and identify on the fly,
* `description` - defined in the tables below,
* `message` - generated depending on the problem and provides more information.
```